proposals/policies/async-decision.md

```diff
@@ -0,0 +1,49 @@
+The scope of this policy is limited to individual documents or deliverables where the editors are the editors of the document or deliverable in scope.
+
+This proposal was originally [submitted](https://lists.w3.org/Archives/Public/public-wot-wg/2021Aug/0006.html) to the public-wot-wg mailing list on 3rd August 2021 and followed up by @benfrancis at https://github.com/w3c/wot/pull/1005/files.
+This version extends it by taking the reviews into account, while moving the problem and notes into an Appendix section.
+
+## Proposed Solution
+
+### Overall Idea
+
+It is proposed to use GitHub's built-in [code review tools](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) to formally review pull requests without having to wait for a decision in a synchronous weekly call.
+This form of asynchronous decision process specifies how to conduct asynchronous decisions that are already written in the Working Group's existing [Decision Policy](https://www.w3.org/2020/01/wot-wg-charter.html#decisions).
+
+### Non-normative Changes
+
+- For non-normative or editorial changes to the wording of a specification or its supporting documents, a code review approval from a single Editor of the specification is sufficient to merge a pull request.
+
+### Normative Changes
+
+- For normative or breaking changes to a specification, approval is required by at least two Editors of a specification that are not from the same organization. Editors should use GitHub's code review tool to either:
+  - Approve the pull request (approval may be conditional on making some small changes like fixing typos before landing, noted in a comment)
+  - Request changes to the pull request, explained in a comment
+  - Comment to say that they have no strong opinion or that they abstain, deferring to the opinion of other editors
+- A pull request can be landed once two editors have either provided their approval or deferred to the other editors.
+- Editors are expected to provide formal reviews, but all members (and non-members) may contribute to the public discussion on GitHub and Editors are responsible for determining consensus. Reviews may be requested from other interested members in order to ascertain consensus.
+- If there are request for changes from other members, they should be taken into account.
+- If a consensus can not be reached by the editors asynchronously where the Pull Request discussion thread goes long (more than 10 comments) without making progress, then a synchronous discussion in a web conference is needed to arrive at a consensus. If a unanimous decision can not be reached, then Chairs may call for a group decision to resolve a deadlock in line with the Working Group's existing [Decision Policy](https://www.w3.org/2020/01/wot-wg-charter.html#decisions).
```

Comment:
more than 10 comments for estimating long discussions is a little bit vague. For example, 10 people commenting about the importance of the changes and saying that they like it, is not really a "long discussion" is just 10 people supporting the changes. Moreover, in one review people can create serveral review comments, are those counted?

Reply:
I had some issues about this as well and was expecting some feedback. I think the TF moderators should identify long and deadlock discussions. How they identify that can be subjective or we agree on some metric but I think it is too difficult to find an objective metric. We can turn it into a more vague statement.

Reply:
Some metrics that we can advise editors to use: 

1. **Number of Comments**: A long and complex discussion often involves multiple participants sharing their thoughts and opinions. Look for a high number of comments on the PR. You can see the comment count on the PR's page.

2. **Comment Depth**: Check if the comments have nested replies. This indicates that the discussion has branched off into sub-conversations, making it more complex.

3. **Comment Length**: Lengthy comments often suggest a detailed and in-depth discussion. Look for comments that span multiple paragraphs or contain extensive explanations.

4. **Time Span**: If the discussion has been ongoing for a significant period, it might indicate complexity. A PR with a long and complex discussion is likely to have active comments spanning days, weeks, or even months.

5. **Emotion and Tone**: Complex discussions can sometimes involve emotional or heated exchanges. Look for strong language, disagreements, or differing opinions expressed in the comments.

5. **Reviewers' Involvement**: If multiple reviewers are actively engaged in the discussion, it suggests a complex review process. This involvement can be seen through their comments, requests for changes, or approval/disapproval of the PR.

My suggestion is to add all/or some of these points in the notes and refer to them in line 29.

Reply:
I have copied all the metrics into an appendix and referred to it from the main text.